target/linux/lantiq/patches-5.10/0720-v5.18-net-phy-at803x-move-page-selection-fix-to-config_init.patch

```diff
@@ -0,0 +1,85 @@
+From 4f3a00c7f5b2cfe4e127fd3fe49b55e1b318c01f Mon Sep 17 00:00:00 2001
+From: Robert Hancock <robert.hancock@calian.com>
+Date: Tue, 25 Jan 2022 10:54:08 -0600
+Subject: [PATCH] net: phy: at803x: move page selection fix to config_init
```

Comment:
I would prefer if you move the backported patches to target/linux/generic/backport-5.15/ and target/linux/generic/backport-5.10.